hack/integration_test/containerfiles/2.containerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+COPY --from=localhost/capo-builder/base:latest /usr/bin/oras /usr/bin/oras
+# The next line doesn't seem to be reflected in the outcome yet
+COPY go.sum /opt/go.sum
```

Comment:
This requires attention. I have found out that files copied form the filesystem are not reflected anywhere with Capo. Is this expected or should this be addressed?

Reply:
Hmm, I tried your containerfile locally and I still didn't see any packages produced :thinking: (Also I had to adjust `fedora:latest` to `registry.fedoraproject.org/fedora:latest` to actually pass for some reason)